daisy_integration_tests/debian_11_worker.wf.json

```
@@ -0,0 +1,42 @@
+{
+  "Name": "debian-11-worker-import-test",
+  "Vars": {
+    "family_tag": {
+      "Value": "debian-11",
```

I would add a worker suffix to the family, as it is not a general Debian image.
```suggestion
      "Value": "debian-11-worker",
```